tests/api_test.py

```diff
@@ -159,6 +160,26 @@ def test_create_newdle_participant_email_sending(flask_client, dummy_uid, mail_q
     assert resp.status_code == 200
 
 
+def test_get_busy_times(flask_client, dummy_uid):
+    _get_busy_times = api._get_busy_times
+    api._get_busy_times = Mock(return_value={})
```

Comment:
Please don't do manual monkeypatching. You can use pytest's [monkeypatch](https://docs.pytest.org/en/stable/monkeypatch.html) fixture. See `test_code_generation` for an example where it's used with mocks

Reply:
Thanks for the suggestion! This is definitely the better way to do it 👍